### PR TITLE
Add test for module, utils, and config

### PR DESCRIPTION
**Pull request by MrLYG**

### CODE REVIEW

File: tests/modules/test_summary_logger.py [hunk 1]
```diff
@@ -261,6 +260,16 @@ def test_results_json(tmp_path: Path):
     pass
 
 
+def test_add_scalar_with_none_value(tmp_path: Path):
+    dashboard = TensorboardLogger(tmp_path.joinpath("tensorboard"))
+    dashboard.add_scalar('test_scalar', None, 1)
```
Comment (fostiropoulos) [inline tests/modules/test_summary_logger.py]: missing assert

File: tests/utils/test_progress_bar.py [hunk 2]
```diff
@@ -134,6 +139,38 @@ def test_remote(i, progress_bar: RemoteProgressBar):
         time.sleep(random.random() / 10)
 
 
+def test_in_notebook():
+    with patch.dict('sys.modules', {'IPython': None}):
+        assert in_notebook() == False
+
+
+def test_get_last_line():
```
Comment (fostiropoulos) [inline tests/utils/test_progress_bar.py]: you should use `tmp_path` fixture.

**Review by fostiropoulos — CHANGES_REQUESTED** [review]
There are missing asserts in some places. In others it is not clear whether for e.g. `math.isnan` is the expected behavior. Can you please add comment on those on the reasoning of the assert ?

**Comment by MrLYG** [comment]
> In others it is not clear whether for e.g. `math.isnan` is the expected behavior

@fostiropoulos  For some functions that return None at some conditions, I want to check if that function will return None at given conditions. And also we can not use `assert value == None` when value = None in python.

So I used the `math.isnan`

**Comment by fostiropoulos** [comment]
Closing this PR as the changes should be merged to `v0.0.1-mp`

**Pull request closed without merging**